modules/hotmaps/1.0/envs/bcftools.yaml

```diff
@@ -0,0 +1,172 @@
+name: vcf2maf
```

Comment:
can we reuse the existing vcf2maf or the one in that module is missing something?

Reply:
The vcf2maf in this env is essentially the same, the only difference is that it contains an updated bcftools version for using "bcftools norm" command. I will try to create a new environment with only the updated bcftools version and use the existing vcf2maf environment for the vcf2maf steps and run tests to make sure it works